platform/ios/INSTALL.md

```diff
@@ -1,6 +1,8 @@
-# Integrating custom builds of the Mapbox iOS SDK into your application
+# Integrating the Mapbox iOS SDK into your application
 
-This document explains how to build a development version of Mapbox iOS SDK for use in your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://mapbox.com/ios-sdk).
+This document explains how to build a _development_ version of Mapbox iOS SDK for use in your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://mapbox.com/ios-sdk) for download links and installation instructions.
+
+## Building and integrating the Mapbox iOS SDK into your application
```

Comment:
This seems redundant to the title up top. Better to promote the rest of the headings by one level.